tests/test_acl.rs

```diff
@@ -0,0 +1,159 @@
+#![cfg(feature = "acl")]
```

Comment:
Given that you have the feature configuration set up correctly, the test should pass CI if we choose to make ACL a feature that's not included by default.